src/callbacks/liquidity/BaselineV2/BaselineAxisLaunch.sol

```diff
@@ -294,36 +327,6 @@ contract BaselineAxisLaunch is BaseCallback, Policy, Owned {
             lotId_, seller_, baseToken_, quoteToken_, capacity_, prefund_, cbData.allowlistParams
         );
 
-        // Calculate the initial active tick from the auction price without rounding
```

Comment:
One of the purposes of this code block was to detect if the Baseline system had been deployed with an active tick different to the auction's fixed price. Is there a scenario where that would be desired?

If the fixed price tick and the Baseline pool's active tick are different, the reserves and BPOOL tokens will be in a different ratio to what the fixed price tick would result in. Isn't that undesirable?